lib/animina_web/live/auth_live/user_auth.ex

```diff
@@ -114,11 +97,28 @@ defmodule AniminaWeb.LiveUserAuth do
     end
   end
 
-  def is_multiple_of_ten(streak) when is_integer(streak) do
-    rem(streak, 10) == 0
+  defp add_daily_points_for_user(_user, _points, _) do
   end
 
-  defp add_daily_points_for_user(_user, _points, _) do
+  def check_if_user_has_daily_bonus_added_for_the_day(user_id) do
```

Comment:
Please don't mix multiple topics into the same PR. But leave it for now.

Reply:
Okay